devdocs/settingUp.md

```diff
@@ -29,7 +29,7 @@ Important: When a version is specified, please install that version instead of t
    * Text encoding: Go to `Window → Preferences → General → Workspace`, change the 
    `Text file encoding` setting from `Default` to `Other: UTF-8`.
    * JRE: Go to `Windows → Preferences → Java → Installed JRE` and ensure a 
-   JDK (not a JRE) is selected(Use a Java 7 JDK, as recommended by GAE). One of the items in the [Troubleshooting help]
+   JDK (not a JRE) is selected. One of the items in the [Troubleshooting help]
```

Comment:
Remove changes not related to this issue. You should merge master into your branch, as this file has been modified in master.